### PR TITLE
Make changes to ug

### LINKED ISSUE #134: [PE-D][Tester E] Invalid JSON File Not Reset
Edit the data file such that the name field is invalid.

The data is not loaded which is correct.

But the data file is not reset to empty.

When I close or exit the application, the corrupted data is still in the json file.

But under the Editing Data File section of the user guide, the data file supposed to be emptied.

-------------
Labels: `severity.Low` `type.FunctionalityBug`
original: nyankoclaws/ped#3

### PR DESCRIPTION
**Pull request by kailashgautham**
Closes #134

### CODE REVIEW

**Review by teojunda — APPROVED** [review]
LGTM!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -113,7 +113,7 @@ You can use the [`ma`](#marking-attendance-ma) command to mark a student as pres
 
 **Tip:**
 Use negative marking to quickly take attendance for a class. Since all students are initially assumed to be present,
-you can quickly the attendace of an entire class by only marking the absentees as absent.
+you can quickly the attendance of an entire class by only marking the absentees as absent.
```
Comment (hiivan) [inline docs/UserGuide.md:116]: "you can quickly TAKE/MARK the attendance"

Comment (kailashgautham) [reply docs/UserGuide.md]: Fixed! Please review!

please? pretty please? pretty please with a cherry? pretty please with a cherry and cream? pretty please with a cherry, cream, and sprinkles? pretty please with a cherry, cream, sprinkles and chocolate sauce? pretty please with a cherry, cream, sprinkles, chocolate sauce and nuts? pretty please with a cherry, cream, sprinkles, chocolate sauce, nuts and oreos? pretty please with a cherry, cream, sprinkles, chocolate sauce, nuts, oreos and caramel? pretty please with a cherry, cream, sprinkles, chocolate sauce, nuts, oreos, caramel and gold flakes? pretty please with a cherry, cream. sprinkles, chocolate sauce, nuts, oreos, caramel, gold flakes and a 50 foot tall ice sculpture?

**Review by g-tejas — APPROVED** [review]
LGTM!

**Pull request merged**